processing file asmFiles/ass.asm (1 of 21)
processing file asmFiles/branch.asm (2 of 21)
processing file asmFiles/count_days.asm (3 of 21)
processing file asmFiles/fib.asm (4 of 21)
processing file asmFiles/halt.asm (5 of 21)
processing file asmFiles/ihit_b4_dhit.asm (6 of 21)
processing file asmFiles/ihit_n_dhit.asm (7 of 21)
processing file asmFiles/jump.asm (8 of 21)
processing file asmFiles/mergesort.asm (9 of 21)
--- memsim.hex	2019-10-24 02:45:59.299154781 -0400
+++ memcpu.hex	2019-10-24 02:45:59.271153947 -0400
@@ -81,134 +81,83 @@
 :040050000800004A5A
 :0400510003E00008C0
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
-:040C40000000057437
-:043FFE0000000500BA
+:0400C1000000005AE1
+:0400C20000000051E9
+:0400C3000000003306
+:0400C400000000191F
+:0400C50000000050E7
+:0400C600000000290D
+:0400C700000000161F
+:0400C800000000151F
+:0400C9000000000C27
+:0400CA000000003EF4
+:0400CB000000004BE6
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000051DD
+:0400CF000000004DE0
+:0400D0000000001616
+:0400D1000000000B20
+:0400D2000000001D0D
+:0400D3000000000722
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
+:040172000000001079
+:040173000000003256
+:040174000000000780
+:040175000000004343
+:040176000000006421
+:040177000000036021
+:04017800000003641C
+:040179000000036817
+:04017A000000036C12
+:04017B00000003700D
+:04017C000000037408
+:04017D000000037803
+:04017E000000037CFE
+:04017F0000000380F9
+:040C40000000031D90
 :00000001FF
processing file asmFiles/mult.asm (10 of 21)
processing file asmFiles/mult_procedure.asm (11 of 21)
processing file asmFiles/raw.asm (12 of 21)
processing file asmFiles/search.asm (13 of 21)
processing file asmFiles/simple.asm (14 of 21)
processing file asmFiles/simple_bne.asm (15 of 21)
processing file asmFiles/simple_raw.asm (16 of 21)
processing file asmFiles/test.branchjump1.asm (17 of 21)
processing file asmFiles/test.ldst1.asm (18 of 21)
processing file asmFiles/test.ldst2.asm (19 of 21)
processing file asmFiles/test.loadstore.asm (20 of 21)
processing file asmFiles/test.rtype.asm (21 of 21)
